bedrock/firefox/templates/firefox/ios.html

```diff
@@ -122,6 +124,38 @@
   </div>{#-- /.container --#}
 </section>{#-- /#sync --#}
 
+<section class="section section-privacy" id="privacy">
+  <div class="container">
+    <h2 class="heading">{{ _('Keep your private info private') }}</h2>
+
+    <p class="lead">
+      {{ _('We believe your personal information belongs to you.') }}
+      {{ _('Firefox for iOS is filled with features to help keep it that way. ') }}
+    </p>
+
+    <ul>
+      <li id="private-browsing">
+      {% if waffle.switch('tracking-protection') %}
```

Comment:
When I try in Aurora 1.1 for iOS, creating a new private tab says “Firefox won't remember any of your history or cookies, but bookmarks will be saved.” Doesn't seem to mention anything about trackers currently. If this isn't going to change, it would effect the line of copy below as well as this headline.